### DISCUSSION TITLE
Why Ollama Embeddings status is Deprecated

### DISCUSSION THREAD

**Discussion by RogerDeng** [n4ze3m/dialoqbase#226] [Q&A] [answered]
I saw a "Ollama Embeddings" in Embedding model list. But when I create a new bot.  The list show "Ollama Embeddings（Deprecated）" . How to solve this problem ? Thanks..

**Comment by n4ze3m** [comment, marked as answer]
Hey, in the older version of Dialoqbase, in order to use '_Ollama embedding_,' you needed to specify model name in environment variables, which was really hard to manage. 

Now, we have added the ability to add Ollama models on the model settings page.Which is why the old method is deprecated in favor of the new one :).

In order to add ollama models check out this example

https://twitter.com/dialoqbase/status/1762900025968050595?t=MLetgRNK1LJXIM4JWFsckg&s=19
